lib/auxiliary/time_helper.rb

```diff
@@ -26,7 +26,7 @@ def time_diff(start, finish)
     end
 
     def timestamp
```

Comment:
Naming this timestamp will collide with `Project#timestamp` attr_reader so it's better to avoid.